plugins/prefixIds.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+exports.type = 'perItem';
+
+exports.active = false;
+
+exports.params = {
+    delim: '__'
+};
+
+exports.description = 'prefix IDs';
+
+
+var path      = require('path'),
```

Comment:
very simple https://github.com/caub/svgo/tree/browser (see dist/ for the bundle), and practical because svgo run fast

 webpack will complain about require('fs') or require('path'), maybe it's possible, but didn't check yet